evmdis/main.go

```diff
@@ -5,29 +5,96 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-
-	"github.com/Arachnid/evmdis"
+	"log"
+	"strconv"
+	"flag"
+	"github.com/soad003/evmdis"
```

Comment:
sure will change that! Is there a better way to do this import because as soon as i fork the project this reference is wrong?